[CI] DateHistogramGroupSourceTests.testBackwardsSerialization72 failure

**Build scan**: https://gradle-enterprise.elastic.co/s/qqcvgqx5jompo

**Repro line**:

```
./gradlew ':x-pack:plugin:core:test' --tests "org.elasticsearch.xpack.core.transform.transforms.pivot.DateHistogramGroupSourceTests.testBackwardsSerialization72" -Dtests.seed=70E39AC50677B868
```

**Reproduces locally?**: Yes (with the random seed)

**Failure excerpt**:

```
java.lang.AssertionError: expected:<{
      "field" : "yeOfWZiq",
      "script" : {
        "id" : "qXvdh"
      },
      "calendar_interval" : "1M"
    }> but was:<{
      "field" : "yeOfWZiq",
      "calendar_interval" : "1M"
    }>
        at __randomizedtesting.SeedInfo.seed([70E39AC50677B868:467188109FBD3A3A]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.elasticsearch.xpack.core.transform.transforms.pivot.DateHistogramGroupSourceTests.testBackwardsSerialization72(DateHistogramGroupSourceTests.java:75)
```

--- a/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/pivot/DateHistogramGroupSourceTests.java
+++ b/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/pivot/DateHistogramGroupSourceTests.java
@@ -59,11 +59,11 @@ public class DateHistogramGroupSourceTests extends AbstractSerializingTestCase<D
 
         return dateHistogramGroupSource;
     }
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/60464")
+
     public void testBackwardsSerialization72() throws IOException {
         // version 7.7 introduced scripts, so test before that
         DateHistogramGroupSource groupSource = randomDateHistogramGroupSource(
-            VersionUtils.randomVersionBetween(random(), Version.V_7_3_0, Version.V_7_7_0)
+            VersionUtils.randomVersionBetween(random(), Version.V_7_3_0, Version.V_7_6_2)
         );
 
         try (BytesStreamOutput output = new BytesStreamOutput()) {

